src/app/pages/large-display/large-display.component.ts

```diff
@@ -70,7 +67,7 @@ export class LargeDisplayComponent implements OnInit {
   toggleTimerRunning() {
     this.timerService.running$.pipe(take(1)).subscribe(running => {
       if (running === true) {
-        this.teacherRoundService.pause();
+        this.teacherRoundService.addHostRoundEvent('Pause');
       }
```

Comment:
I'd suggest we have two events, `pause` and `play` or something and do something like this:
```ts
this.teacherRoundService.addHostRoundEvent(running ? EventType.Play : EventType.Pause);
```

But also this shouldn't be done here on the button because that won't capture all pause/play events. I think it should probably happen in the `TeacherRoundService` where the paused observable is subscribed to, though there it will also add a single initial event at the start of the round, but that's probably ok.

Reply:
Alternatively we could have just a single event type for changes in `running` state and have additional data on the event for if running became true or false. But that would probably be more complicated.